set.mm

```diff
@@ -548850,6 +548900,515 @@ class of symmetric relations as well (like the elements of the class of
     ( crefrels csymrels cin wcel wrefrel wsymrel elin elrefrelsrel elsymrelsrel
     wa anbi12d syl5bb ) ACDEFACFZADFZLABFZAGZAHZLACDIQORPSABJABKMN $.
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Transitivity
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $( Define the class of all transitive sets (vs. the transitive class defined
+     in ~ df-tr ).  It is used only by ~ df-trrels .
+
+     Note the similarity of the definitions of ~ df-refs , ~ df-syms and
+     ~ df-trs .  (Contributed by Peter Mazsa, 17-Jul-2021.) $)
+  df-trs $a |- Trs = { x |
+      ( ( x i^i ( dom x X. ran x ) ) o. ( x i^i ( dom x X. ran x ) ) ) _S
+        ( x i^i ( dom x X. ran x ) ) } $.
+
+  $( Define the class of transitive relations.  The element of the class of
+     transitive relations and the transitive relation predicate are the same
+     when ` R ` is a set, cf. ~ eltrrelsrel .  Alternate definitions are
```

Comment:
For sets, being an element of the class of.... is equivalent to satisfying the ... predicate.
(idem below)

Reply:
done